cookbooks/plume_2D_chunk/doc/plume.tex

```diff
@@ -0,0 +1,98 @@
+Note that in the original publication the portion of the inner boundary that is not the patch is also insulating. By default, \aspect{} cannot accommodate two different boundary condition types on one boundary. So, we will prescribe plausible temperatures along the whole boundary, instead: $T\textsubscript{surf}=0\si{\celsius}$, $\Delta T=3000\si{\celsius}$, and $T\textsubscript{cmb}=2750\si{\celsius}$ (see for instance Steinberger \& Calderwood \cite{stca06}).
+
+The domain contains a single fluid described by the `visco plastic' material model, with thermal expansion $\alpha=3\times 10^{-5}~\si{\per\kelvin}$, heat capacity $C_p=1250~\si{\joule\per\kelvin}$, thermal diffusivity $\kappa=5.5\times 10^{-7}~\si{\metre\per\second}$ (corresponding to the thermal conductivity value  $k=\kappa \rho_0 C_p=2.25~\si{\watt\per\metre\per\kelvin}$), reference temperature $T_0=1023~\si{\kelvin}$, reference density  $\rho_0=3250~\si{\kg\per\cubic\metre}$, viscosity $\eta_0=1.25\times 10^{23}$. Gravity is constant throughout the mantle with $g=9.81~\si{\metre\per\square\second}$.
+With these parameters we find that the dimensionless Rayleigh number is 
+\begin{equation}
+Ra 
+=\frac{\rho_0 g \alpha \Delta T (R\textsubscript{outer}-R\textsubscript{inner})^3}{\kappa \eta_0}
+=10^6.
+\end{equation}
+In Kellogg \& King (1997), the dimensionless viscosity is temperature-dependent and defined as:
+\[
+\eta'(T')
+=\eta_0' \exp\left[ \frac{E}{R \Delta T} \left( \frac{1}{T'+T_0'} -\frac{1}{1+T_0'}  \right)   \right],
+%=\eta_0' \exp\left[ \frac{E}{R} \left( \frac{1}{T' \Delta T  +t_0 \Delta T} -\frac{1}{\Delta T +t_0 \Delta T}  \right)   \right]
+\]
+where $\eta_0'=1$ is the dimensionless viscosity, $E$ is the activation energy, and $R$ is the universal gas constant. The dimensionless temperature $T_0'$ is the surface temperature $T_{surf}$ divided by the temperature drop across the shell $\Delta T$. 
+Assuming that the authors used the common relationship	 
+\[
+T'=\frac{T-T\textsubscript{surf}}{T\textsubscript{patch}-T\textsubscript{surf}}= \frac{T-T\textsubscript{surf}}{\Delta T}.
+\]
+then multiplying the equation above by $\eta_0$, it follows that:
+\[
+\eta(T)
+=\eta_0 \exp\left[ \frac{E}{R} \left( \frac{1}{T-T\textsubscript{surf}  + T\textsubscript{surf}} 
+-\frac{1}{T\textsubscript{patch}-T\textsubscript{surf} + T\textsubscript{surf}}  \right)   \right]
+=\eta_0 \exp\left[ \frac{E}{R} \left( \frac{1}{T} -\frac{1}{T\textsubscript{patch}}\right) \right]
+\]
+so that $\eta(T_{patch})=\eta_0$.
+%
+Kellogg \& King (1997) investigated three cases:
+$E/R \Delta T = \{0,0.25328,3\}$. Setting $R=8.31~\si{\joule\per\kelvin\per\mol}$ and $\Delta T=3000\si{\kelvin}$, the activation energy becomes $E=\{ 0, 6317.6 , 74829.6 \}\si{\joule}$, lower than typical values (above 200~\si{kJ}, see for example Karato \& Wu \cite{KW93}).
```

Comment:
The unit of E should be joule per mol.

Reply:
I must have been real tired the other night ...